packages/facility-server/app/routes/apiv1/patient/patientInvoice.js

```diff
@@ -2,12 +2,44 @@ import { INVOICE_STATUSES } from '@tamanu/constants';
 import { getInvoiceSummary } from '@tamanu/shared/utils/invoice';
 import express from 'express';
 import asyncHandler from 'express-async-handler';
+import { getInsurancePlanItems } from '../invoice/getInsurancePlanItems';
 
 export const patientInvoiceRoutes = express.Router();
 
 const encounterOrderByKeys = ['encounterType'];
 const invoiceOrderByKeys = ['date', 'displayId', 'status'];
 
+// Shared function to hydrate invoices with price list associations
+async function hydrateInvoices(invoiceRecords, models) {
```

Comment:
For each invoice record, we need to include the price list items and insurance items